cpp/daal/src/algorithms/k_nearest_neighbors/bf_knn_classification_model_impl.h

```diff
@@ -1,4 +1,4 @@
-/* file: bf_knn_classification_model_ucapi_impl.h */
+/* file: bf_knn_classification_model_impl.h */
```

Comment:
What are the rules on this? This file was included by several non-sycl files so could not delete entirely, instead removed context and cpu/gpu conditional. To me since there is no new logic just slight revision of conditionals it wouldn't warrant updated copyright but I don't know what the guidelines are for this